site/en/blog/third-party-cookie-deprecation-trial/index.md

```diff
@@ -0,0 +1,221 @@
+
+Top-level sites that use third parties that rely on cookies **do not** need to sign up for this deprecation trial. You should audit the third-party cookies used within your site and contact your third-party providers to ensure they are prepared for the deprecation.
+
+
+## Eligibility criteria and review process
+
+This deprecation trial differs from previous trials with the introduction of a review and approval process for participation. This is to strike a balance between improving privacy for people on the web, while still enabling the services they depend on to request extra time to migrate if necessary.
+
+The principles guiding this deprecation trial are: 
+
+* **Preserving user-critical functionality:** This deprecation trial is intended for third-party providers that demonstrate functional breakage in user journeys.
+* **Limiting user tracking:** The deprecation trial is not intended to support cross-site tracking for advertising purposes, and as such third-party embeds and services used for advertising are not eligible.
+
+The ineligibility of advertising use cases will also help to ensure the deprecation trial does not interfere with the [industry testing planned for the start of 2024 as described by the Competition and Markets Authority](https://www.gov.uk/cma-cases/investigation-into-googles-privacy-sandbox-browser-changes#industry-testing). This includes advertising-related domains that are also used for non-advertising purposes.
+
+Chromewill initially work with Disconnect.me, an industry leader in internet privacy, and implement Disconnect.me's tracker protection lists to identify the scripts and domains categorized as advertising. Disconnect is already used by other browsers for similar purposes on the web.
+
+We will apply the following process for registration requests:
+
+1. If the third-party origin matches a known advertising domain, including if the origin matches an entry on the disconnect.me [advertising list](https://github.com/disconnectme/disconnect-tracking-protection/blob/master/services.json#L1535C11-L1535C11), then the registration request will be rejected. In general, if a domain is considered an advertising domain, subdomains of that domain will also be considered advertising List entries scoped to a particular pathname, however, will have the pathname removed and match to the exact origin in the entry.
+2. Steps to reproduce a broken user-facing experience must be provided. In particular, this should be an experience for the user operating the device where the cookie is stored, and not a user performing later analysis of data. If we cannot validate a broken user experience then the registration request will be rejected.
+3. Otherwise the registration request will be approved.
```

Comment:
Surely point 3 isn't a logical third step? I would say to not use a numbered list here, it does not seem to be a sequence of instructions. It can be two bullet points and then this line as a conclusion.

Reply:
Converted to bullets, but kept them all in the list just for keeping them visually close and emphasised the outcome at each step, i.e. if it's this: rejected, if it's that: rejected, otherwise: approved.